setup.py

```diff
@@ -16,7 +16,7 @@
 from setuptools import find_packages
 
 extras = {}
-extras["quality"] = ["black ~= 23.1", "ruff >= 0.0.241", "hf-doc-builder >= 0.3.0"]
+extras["quality"] = ["black ~= 23.1", "ruff >= 0.0.241", "hf-doc-builder >= 0.3.0", "urllib3 <= 2.0.0"]
```

Comment:
```suggestion
extras["quality"] = ["black ~= 23.1", "ruff >= 0.0.241", "hf-doc-builder >= 0.3.0", "urllib3 < 2.0.0"]
```
2.0.0 was yanked but technically we don't want it either.